histomicstk/cli/NucleiDetection/NucleiDetection.py

```diff
@@ -17,20 +18,53 @@
 logging.basicConfig(level=logging.CRITICAL)
 
 
+def read_input_image(args, process_whole_image=False):
+    print('\n>> Reading input image ... \n')
+
+    ts = large_image.getTileSource(args.inputImageFile, style=args.style)
+
+    ts_metadata = ts.getMetadata()
+
+    # print(json.dumps(ts_metadata, indent=2))
+
+    is_wsi = ts_metadata['magnification'] is not None
+    # TODO - Remove this checkpoint
+    print('\n >> is it wsi ? {}, is it whole image ? {}'.format(is_wsi, process_whole_image))
+
+    return ts, is_wsi
+
+
+def image_inversion_flag_setter(args=None):
+    invert_image = False
+    if args.ImageInversionForm == "Yes":
+        invert_image = True
+    if args.ImageInversionForm == "No" or args.ImageInversionForm == "unspecified":
```

Comment:
I think rather than "unspecified", this should have a "default" mode which, when a single band image will invert and an RGB image would not invert.  By making that the default, for the case we are pursuing the inversion will happen on our fluorescence channel frame but not on our H&E images.

Reply:
I have added the default mode to frame and style. As you mentioned before, If it's a single-channel image the default option will invert the image and an RGB image would not invert. I have tested them on sample RGB and single-channel images.